docs/refs.bib

```diff
@@ -1,3 +1,113 @@
+@article{sauter,
+title = {Geometric formulas for system codes including the effect of negative triangularity},
+journal = {Fusion Engineering and Design},
+volume = {112},
+pages = {633-645},
+year = {2016},
+issn = {0920-3796},
+doi = {https://doi.org/10.1016/j.fusengdes.2016.04.033},
+url = {https://www.sciencedirect.com/science/article/pii/S0920379616303234},
```

Comment:
Just the doi is enough otherwise it renders as multiple fields, see below
```suggestion
doi = {https://doi.org/10.1016/j.fusengdes.2016.04.033},
```

<img width="704" alt="image" src="https://github.com/cfs-energy/cfspopcon/assets/11351671/731c76f6-6a76-43a0-8c8a-086229d16dd2">